docs/BUILD.md

```diff
@@ -298,30 +300,30 @@ The following custom packages are used:
 
 * kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. This is primarily due to kernel modules and drivers, especially on arm, as well as Xen requirements.
 * `init` packages:
-    * `zededa/grub` - 
-    * `zededa/devices-trees` - 
-    * `zededa/fw` - 
-    * `zededa/xen` - 
-    * `zededa/gpt-tools` - 
-    * `zededa/dom0-ztools` - 
+    * `zededa/grub` - CoreOS inspired GRUB required to enable CoreOS-style dual partition upgrades.
+    * `zededa/devices-trees` - device trees for all the ARM platforms that EVE supports.
+    * `zededa/fw` - various firmware required for device drivers.
+    * `zededa/xen` - a single Xen binary required to boot EVE.
+    * `zededa/gpt-tools` - ChromiumOS inspired tools and sgdisk required to enable CoreOS-style dual partition upgrades. 
+    * `zededa/dom0-ztools` - catch-all containers for tools helpful in developing and debugging EVE. 
 * `onboot` packages:
-    * `zededa/rngd` - custom `zededa/rngd` package, rather than the standard linuxkit one. _Why?_
+    * `zededa/rngd` - custom `zededa/rngd` package, rather than the standard linuxkit one. This micro-fork accommodates the [following hack](https://github.com/zededa/zenbuild/blob/master/pkg/rngd/cmd/rngd/rng_linux_arm64.go) which provides some semblance of seeding randomness on ARM. Without this HiKey board won't boot. 
 * `services` packages:
-    * `zededa/wwan` - WWAN drivers and software. LTE? 3G? 2G?
-    * `zededa/wlan` - WLAN drivers and software.
-    * `zededa/guacd` - 
+    * `zededa/wwan` - WWAN drivers and software. LTE/3G/2G. Mostly experimental.
+    * `zededa/wlan` - WLAN drivers and software. Currently a glorified wrapper around wpa_supplicant.
+    * `zededa/guacd` - [Apache Guacamole service](http://guacamole.apache.org/) that provides console and VDI services to running VMs and containers. 
     * `zededa/zedctr` - a "catch-all" package for EVE tools; see below.
 
 **Installer**
 
-* kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. _Why?_
+* kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. This is required since EVE's installer has a double duty as a verification tool for whether hardware can support running Xen. We do this by booting Xen (see below) with the same kernel that the actual EVE deployment will use.
```

Comment:
Not sure I get this. Is this something installation time or run time? What does the boot process look like between kernel and xen, and how is it different from "normal" xen+kernel?